### DISCUSSION TITLE
Using pins other than GPIO 25, 26 for audio output into MAX98357a

### DISCUSSION THREAD

**Discussion by ckikkeri1** [pschatzmann/ESP32-A2DP#411] [Q&A] [answered]
Hi, is it possible to get proper audio output using pins other than internal DAC pins of GPIO 25 and 26? I am using MAX98357a which says it has a DAC, so is it possible to use some other ESP pins' analog output and then have the amplifier convert it?

Great library btw :)

**Comment by pschatzmann** [comment, marked as answer]
The Readme describes how to connect an external DAC and there is a separate chapter in the Wiki, so I don't understand why you even ask this question or am I missing something?
